blockchains/ethereum/assets/0x66a0f676479Cee1d7373f3DC2e2952778BfF5bd6/info.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
@Cryptocool1 please put this file back and merge again, this is my token's correct info.json and you deleted it incorrectly.

Reply:
I paid 250 TWT to update my logo and info, and you've deleted the info.  Please correct this ASAP.

Reply:
@catenocrypt @zachzwei can someone please assist?

Reply:
@Cryptocool1 this ain't cool at all! Our team submits the correct token info, paid the price and you've deleted our json file with all the info! Is this some kind of a joke! We want then a refund and you can revert this PR!!!!!